### PR TITLE
Javascript data model

### PR DESCRIPTION
**Pull request by Quintin-Russell**
image

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+var orderHistory = [
+  {
+    orderNumber: '11439416898772232',
+    orderDate: '08/04/2020',
+    currency: 'dollars',
+    listedPrice: 31.55,
+    totalPrice: 34.00,
+    productType: 'book',
```
Comment (BrettAlbright) [inline javascript-data-model/order-history.js]: make sure to update your data model with the `items` array of objects to encapsulate the individual item data in objects: 
```javascript
var orderHis = [
  {
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "book",
      title: "A New Way to Learn"
    }
  ]
  }
]
```

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Great initial submission, small update needed

**Comment by Quintin-Russell** [comment]
modified. ready for review

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**